### ISSUE TITLE
[CAIA A11y Research]: Authentication Experience, My VA (dot indicator usage)

### ISSUE THREAD

**Issue by mattmarino-adhoc** [department-of-veterans-affairs/va.gov-team#73932]
### Who's on your team?

```bash
Team name:Authentication Experience (My VA)
OCTO-DE product owner: Samara Strauss
Product manager: Anastasia(AJ) Jakabcin
Designer: Matt Marino
FE engineer: Allison Liu
Product/team Slack channel: #accountexp-authexp
```

### Do you have an accessibility champion?

No, but we are interested in training one!

### Is anyone on your team interested in becoming an accessibility champion?

Me (Matt Marino)

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

Before the end of February

### Supporting artifacts (if any)

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/my-va#product-outline-my-va)
- [User flows](https://www.figma.com/file/15yOY4VEzitxm5tRMDiAzz/My-VA?type=design&node-id=0-7253&mode=design)
- [Prototype](https://www.figma.com/proto/15yOY4VEzitxm5tRMDiAzz/My-VA?node-id=205-27290&starting-point-node-id=205%3A27290)
- [Figma File](https://www.figma.com/file/15yOY4VEzitxm5tRMDiAzz/My-VA?type=design&node-id=196%3A26639&mode=design&t=3uRwcSw5HvV7vL1R-1)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/onsite-notifications/dot-indicator/research/dot-indicator-research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/onsite-notifications/dot-indicator/research/dot-indicator-convo-guide.md)
- [Screen reader conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/onsite-notifications/dot-indicator/research/dot-indicator-convo-guide-for-screen-readers.md)
- [Epics/issue that may be helpful](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63788)

### Next steps

- [ ] Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [ ] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [ ] CAIA Specialists are assigned to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
- [ ] CAIA Specialists review your conversation guide and research plan
- [ ] Provide specialists your participant details like skill level, operating system, assistive tech used, browser
- [ ] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [ ] CAIA Specialists create a research notes template
- [ ] CAIA Post research, specialists will add accessibility findings to your research folder

**Comment by coforma-terry** [comment]
Assigning @sara-amanda for visibility here - thanks!

**Comment by sara-amanda** [comment]
# Intake Received

Hi, @mattmarino-adhoc ! Just a quick note here to let you know that we received your intake ticket and will be processing it later this week. It normally isn't like this, we have had a sudden influx of studies, taking place prior to yours. 

## Next Steps
- [ ] Intake ticket to be processed, and I can reach out for additional info.
- [ ] Do you happen to have time slots and dates available that I can place holds on our [CAIA A11y Capacity calendar](https://calendar.google.com/calendar/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a%40group.calendar.google.com&ctz=America%2FNew_York), in the meantime?
- [ ] We will get a meeting on the calendar, per your Slack request, below. 

> I'd like to set up a time to talk this sprint about the conversation guide/research plan for assistive tech users.

All the best, 

Sara

cc: @coforma-terry

**Comment by sara-amanda** [comment]
## Ticket Updated
Hi, @mattmarino-adhoc ! It was great talking to you today! Here is an update on your ticket for research. 

### Participant Info

This ticket has been prepped for your future testing. I have created a high-level summary at the top of the ticket, added two tables (one for your participants, so that we can schedule available a11ys) and the other to collect assistive tech. information, so that we can prep for each session. 
- [ ] **Table 1:** [Participants](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73932#:~:text=Table%201%3A%20Participants)
- [ ] **Table 2:** [Technology](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73932#:~:text=and%20Device%20Type.-,Table%202%3A%20Technology,-Toggle%20to%20view)

### Conversation Guide
Additionally, @coforma-jamie has [completed her review of the conversation guide for screen readers.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75143#issuecomment-1922221752) Her feedback is linked in your task list above and here.

### Pilot
A ticket has been generated for the CAIA team to track the pilot work for 2/9/2024 via ticket [75143](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75143). Both @eli-oat and @coforma-jamie appear to be available for that pilot session. 

### Tasks
The task lists for each of us have been created, so that you can see what CAIA is doing, and if you have any outstanding tasks yourself. 

We are looking forward to supporting your study! 

All the best, 

Sara

cc: @coforma-terry

**Comment by sara-amanda** [comment]
## Pilot Postponed
[Access information, relating to rescheduling the pilot in the pilot ticket. ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75143#issuecomment-1936086441)

**Comment by coforma-jamie** [comment]
From @mattmarino-adhoc:

> The dot indicator/slim alert research recruitment effort is now ready for Perigean, a kickoff call should hopefully be happening early next week

Will be on the lookout for dates later next week!

**Comment by sara-amanda** [comment]
Updated ticket Table 1 to reflect shift to March testing.

**Comment by coforma-jamie** [comment]
AT sessions will be held March 11-15, 17-18. @mattmarino-adhoc will update the participant table in this ticket, and invite @sara-amanda to the sessions (we'll assign CAIA a11ys from there).

**Comment by sara-amanda** [comment]
## Table Updates
@mattmarino-adhoc I have updated the [Testing Schedule Table](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73932#:~:text=Toggle%20to%20view%20the%20AT%20testing%20schedule) with a11ys. 
- **Open Slot:**
    - One invite not yet received for 3/11 at 1 p.m. ET @eli-oat @SarahKay8 @EvanAtCoforma or myself will try and fill this slot.
- **Removing**
    - P3 | 3/11/24, 1:00pm – 2:00pm based on [Slack Canvas ](https://dsva.slack.com/docs/T03FECE8V/F06NLT96GNR) link shared with @coforma-jamie 
    - Adding Slack Canvas link to ticket.

**Comment by coforma-jamie** [comment]
This study is complete as of April 1, 2024. @mattmarino-adhoc received CAIA feedback via Mural, Slack, and our [notes document](https://docs.google.com/document/d/1w4Hlf8qSnl3LD8X199-6IFetXujPF1nto9godcK1x-s/edit), and didn't need us to synthesize our findings further.

Thank you for working with us!
